aws/resource_aws_db_option_group.go

```diff
@@ -107,6 +107,10 @@ func resourceAwsDbOptionGroup() *schema.Resource {
 							Elem:     &schema.Schema{Type: schema.TypeString},
 							Set:      schema.HashString,
 						},
+						"version": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
```

Comment:
Can this attribute be retrieved by the AWS API without a user setting it?
If so, this should also have a `Computed: true,` option